keyboards/skyloong/gk980/q1/info.json

```diff
@@ -0,0 +1,80 @@
+{
+    "manufacturer": "skyloong",
+    "maintainer": "NaturalZh",
+    "bootloader": "stm32duino",
+    "build": {
+        "debounce_type": "asym_eager_defer_pk"
+    },
+    "eeprom": {
+        "wear_leveling": {
+            "backing_size": 4096
+        }
+    },
+    "encoder": {
+        "rotary": [
+            {"pin_a": "A1", "pin_b": "A0"},
+            {"pin_a": "A3", "pin_b": "A2", "resolution": 2},
+            {"pin_a": "A4", "pin_b": "B0", "resolution": 2},
+            {"pin_a": "B3", "pin_b": "B1", "resolution": 2},
+            {"pin_a": "B10", "pin_b": "B4", "resolution": 2}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": true,
+        "console": false,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true
+    },
+    "indicators": {
+        "num_lock": "C15",
+        "on_state": 1,
+        "caps_lock": "C14"
+    },
+    "matrix_pins": {
+        "cols": [null, null, null, null, null, null, null, null, null, null, null, null, null, null, null],
+        "rows": ["A8", "A9", "A10", "A15", "B6", "B7","B9"]
+    },
```

Comment:
These should go back in `config.h`.

Reply:
It has been corrected. Would you please review it again? Thanks for your suggestion. @fauxpark